chat.delta.desktop.yml

```diff
@@ -0,0 +1,126 @@
+app-id: chat.delta.desktop
+tags:
```

Comment:
fair enough. is it harmful though?

Reply:
right. so desktop file is somehow generated by electron-builder. I have only extracted it from the Debian package. Touching it is not sustainable.
Do you know how to influence the tags via electron-builder? Otherwise I guess we can leave it for another time.

Reply:
I think tags could be also used in the future to filter out proprietary software from store implementations and programs so I'd rather not pollute it here.